templates/all-actions.twig

```diff
@@ -0,0 +1,42 @@
+{% extends "base.twig" %}
+
+{% block content %}
+	<div class="clearfix"></div>
+	<div class="skewed-overlay"></div>
+
+	<div class="page-content container">
+		{% if query_loop %}
+			{{ query_loop|raw }}
+		{% else %}
+			<h3>{{ __( 'Results', 'planet4-master-theme' ) }}</h3>
+
+			<div class="row">
+				<div class="col-lg-8 multiple-search-result">
+					<ul class="list-unstyled">
+						{% for post in posts %}
+							{% include 'tease-taxonomy-action.twig' %}
+						{% endfor %}
+					</ul>
+				</div>
+			</div>
+
+			{% if posts.pagination.total > 1 %}
+				<div class="row">
+					<div class="col-md-12 col-lg-5 col-xl-5 mt-3">
+						<button
+							data-content=".multiple-search-result .list-unstyled"
+							data-page="1"
+							data-total="{{ posts.pagination.total }}"
+							data-url="{{ url }}"
+							class="load-more-mt btn btn-small btn-secondary mb-5"
+							data-ga-category="Articles List"
```

Comment:
Shouldn't this category be changed? 🤔

Reply:
Same in `templates/tease-taxonomy-action.twig`

Reply:
Not sure about this. Do you mean to change to "Actions List" or to any other category?

Reply:
Yeah I think `Actions List` sounds good 👍 Maybe double-check with Molly though 🤔